core/includes/file.inc

```diff
@@ -2018,6 +2018,64 @@ function file_validate_image_orientation(File $file, $orientation = 0) {
   return $errors;
 }
 
+/**
+ * Validate uploaded SVG files.
+ *
+ * @param File $file
+ *   A file entity.
+ *
+ * @return
+ *   An array. If the file is an SVG and is either not valid or contains
+ *   dangerous content, it will contain an error message.
+ *
+ * @see hook_file_validate()
```

Comment:
The function name below doesn't seem like actually having anything to do with hook_file_validate, though.

Is this supposed to implement the hook?

Reply:
All the other `file_validate_*()` functions in `file.inc` reference ` @see hook_file_validate()`, so I think this one should follow same pattern.

Reply:
I'm going to tentatively close this. I assume it's because `file_validate()` calls all these other validation functions and allow the hook to change the results.